migrations/Version202107161250081888_taoMediaManager.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoMediaManager\migrations;
+
+use oat\taoMediaManager\controller\MediaManager;
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\taoMediaManager\model\classes\user\TaoAssetRoles;
+use oat\tao\scripts\tools\accessControl\SetRolesAccess;
+use oat\tao\scripts\update\OntologyUpdater;
+use oat\tao\model\accessControl\ActionAccessControl;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202107161250081888_taoMediaManager extends AbstractMigration
```

Comment:
Remove always, please. It is not necessary.

```suggestion
final class Version202107161250081888_taoMediaManager extends AbstractMigration
```

Reply:
Done